modules/monitoring-roles-for-monitoring-your-own-services.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="roles-for-monitoring-your-own-services_{context}"]
+= Roles for monitoring your own services
+
+This table lists the cluster roles deployed using the Cluster Monitoring Operator (CMO). Each of these roles can be bound against a specific namespace by the cluster administrator.
+
+.Roles for monitoring your own services
+[options="header", cols="2,5a,5a"]
+|====
+|Role|Permissions embedded by the role|What the role enables
+|`monitoring-rules-view`
+|`get prometheusrules.monitoring.coreos.com`
+|* Reading PrometheusRule custom resources matching the permitted namespace.
+* Getting declared and pending alerts and recording rules from `/api/v1/alerts?namespace=<foo>` and `/api/v1/rules?namespace=<foo>` endpoints matching the permitted namespace.
+
+|`monitoring-rules-edit`
+|`create/edit/delete prometheusrules.monitoring.coreos.com`
+|* Creating, modifying, and deleting PrometheusRule custom resources matching the permitted namespace.
+* Silencing firing alerts from the Alertmanager `/alerts?namespace=<foo>` endpoint matching the permitted namespace.
+
+|`monitoring-edit`
+|`get/create/edit/delete` for these custom resources:
+
+. ServiceMonitor
+. PodMonitor
+. PrometheusRule
+|* Creating new scraping targets for services or pods.
+* Creating new recording or alerting rules.
+|====
+
+// [For 4.6+] Additionally, the `monitoring-workload-config-edit` named role is deployed against the `workload-monitoring-config` configmap resource in the `openshift-monitoring` namespace, which embeds the `get/create/edit/delete` permissions.
```

Comment:
Do we need this now? Also this role has not even been shipped and we have not finalised the name of it :) I would remove this for now.